stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -939,38 +907,47 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     /// - parameter count: The number of bytes to copy.
     public init(bytes: UnsafeRawPointer, count: Int) {
         _backing = _DataStorage(bytes: bytes, length: count)
+        _sliceRange = 0..<count
```

Comment:
the declaration of _sliceRange needs to be initialized with the parameter count not the property count, so the initial value is dependent upon the initialized value of the backing